### PR TITLE
Release v0.28.0

### PR DESCRIPTION
**Pull request by jchengsfx**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +16,25 @@
...
+- Remove deprecated funcs inside the obsreceiver (#3314)
+- Rename internal/otlp files, avoid duplicate initialization (#3317)
+- Remove `obsreport.GRPCServerWithObservabilityEnabled`, enable observability in config (#3315)
+- Remove `obsreport.ProcessorMetricViews`, use `BuildProcessorCustomMetricName` where needed (#3316)
+- Rename `TraceData` to `Traces` in tests (#3325)
+- Remove "Receive" from `obsreport.Receiver` funcs (#3326)
+- Rename internal funcs `push*Data` to `push*` (#3324)
+- Rename `authoidcextension` to `oidcauthextension` (#3247)
+- Remove "Export" from `obsreport.Exporter` funcs (#3333)
+- Hide unnecessary public struct in obsreport (#3353)
```
Comment (dmitryax) [inline CHANGELOG.md:31]: Let's add struct name: `obsreport.StartReceiveOptions`

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +16,25 @@
...
+- Rename `PrwExporter` to `PRWExporter` and `NewPrwExporter` to `NewPRWExporter` (#3246)
+- Avoid exposing OpenCensus reference in public APIs (#3253)
+- Move `config.Parser` to `configparser.Parser` (#3304)
+- Remove deprecated funcs inside the obsreceiver (#3314)
+- Rename internal/otlp files, avoid duplicate initialization (#3317)
+- Remove `obsreport.GRPCServerWithObservabilityEnabled`, enable observability in config (#3315)
+- Remove `obsreport.ProcessorMetricViews`, use `BuildProcessorCustomMetricName` where needed (#3316)
+- Rename `TraceData` to `Traces` in tests (#3325)
+- Remove "Receive" from `obsreport.Receiver` funcs (#3326)
+- Rename internal funcs `push*Data` to `push*` (#3324)
```
Comment (dmitryax) [inline CHANGELOG.md:28]: Let's remove it, no effect on exposed API

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,6 +16,25 @@
...
 - Replace `ReceiverCreateParams` with `ReceiverCreateSettings`. (#3167)
 - Change `batchprocessor` logic to limit data points rather than metrics (#3141)
+- Rename `PrwExporter` to `PRWExporter` and `NewPrwExporter` to `NewPRWExporter` (#3246)
+- Avoid exposing OpenCensus reference in public APIs (#3253)
+- Move `config.Parser` to `configparser.Parser` (#3304)
+- Remove deprecated funcs inside the obsreceiver (#3314)
+- Rename internal/otlp files, avoid duplicate initialization (#3317)
+- Remove `obsreport.GRPCServerWithObservabilityEnabled`, enable observability in config (#3315)
+- Remove `obsreport.ProcessorMetricViews`, use `BuildProcessorCustomMetricName` where needed (#3316)
+- Rename `TraceData` to `Traces` in tests (#3325)
```
Comment (dmitryax) [inline CHANGELOG.md:26]: Let's remove it, just a change to tests, no effect on exposed API

File: CHANGELOG.md [hunk 4]
```diff
@@ -14,6 +16,25 @@
 - Replace `ExporterCreateParams` with `ExporterCreateSettings` (#3164)
 - Replace `ReceiverCreateParams` with `ReceiverCreateSettings`. (#3167)
 - Change `batchprocessor` logic to limit data points rather than metrics (#3141)
+- Rename `PrwExporter` to `PRWExporter` and `NewPrwExporter` to `NewPRWExporter` (#3246)
+- Avoid exposing OpenCensus reference in public APIs (#3253)
+- Move `config.Parser` to `configparser.Parser` (#3304)
+- Remove deprecated funcs inside the obsreceiver (#3314)
+- Rename internal/otlp files, avoid duplicate initialization (#3317)
```
Comment (dmitryax) [inline CHANGELOG.md:23]: Let's remove this line, no effect on exposed API

File: CHANGELOG.md [hunk 5]
```diff
@@ -22,6 +43,27 @@
 - Enable Dependabot for Github Actions (#3312)
 - Remove the proto dependency in `goldendataset` for traces (#3322)
 - Add telemetry for dropped data due to exporter sending queue overflow (#3328)
+- Use `sharedcomponent` package helper for OC receiver (#3228)
```
Comment (dmitryax) [inline CHANGELOG.md:41]: Let's remove this line, internal refactoring

File: CHANGELOG.md [hunk 6]
```diff
@@ -22,6 +43,27 @@
...
+- Add initial implementation of `pdatagrcp` (#3231)
+- Change receiver obsreport helpers pattern to match the Processor/Exporter (#3227)
+- Add model translation and encoding interfaces (#3200)
+- Add `doc.go` files to the Config package (#3284)
```
Comment (dmitryax) [inline CHANGELOG.md:45]: I don't think this change should be mentioned as well

File: CHANGELOG.md [hunk 7]
```diff
@@ -22,6 +43,27 @@
...
+- Add otlpjson as a serializer implementation (#3238)
+- `prometheus` receiver:
+  - Add `createNodeAndResourcePdata` for Prometheus->OTLP pdata (#3139)
+  - Direct metricfamily Prometheus->OTLP (#3145)
+- Improve component package docs (#3305)
```
Comment (dmitryax) [inline CHANGELOG.md:50]: Same here

File: CHANGELOG.md [hunk 8]
```diff
@@ -22,6 +43,27 @@
...
+- Add `componenttest.NewNop*CreateSettings` to simplify tests (#3375)
+- Convert fileexporter to not depend on InternalRep (#3384)
```
Comment (dmitryax) [inline CHANGELOG.md:52]: Same here, just internal refactoring

**Review by tigrannajaryan — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

**Pull request merged**